pkgs/by-name/ya/yandex-music/update.sh

```diff
@@ -8,6 +8,7 @@ REPO="yandex-music-linux"
 URL="https://api.github.com/repos/$OWNER/$REPO"
 RAW="https://raw.githubusercontent.com/$OWNER/$REPO"
 
+pname="yandex-music"
```

Comment:
pname has a semantically different meaning

```suggestion
attrname="yandex-music"
```